esmvaltool/utils/recipe_test_workflow/site/dkrz.cylc

```diff
@@ -0,0 +1,55 @@
+#!jinja2
+[runtime]
+    [[root]]
+        [[[environment]]]
+            CONTAINER_DIR = ${ROSE_DATAC}/container
+            CONTAINER_FILE = esmvaltool.sif
+            CONTAINER_PATH = ${CONTAINER_DIR}/${CONTAINER_FILE}
+            # Warning: fragile assumption of location of esmvaltool code,
+            # issue #3437 contains more details.
+            ESMVALTOOL_DIR = /opt/conda/envs/esmvaltool/lib/python3.11/site-packages
+
+    # COMPUTE provides defaults for computation-heavy tasks.
+    # Specific tasks below override some defaults, e.g. time & memory.
+    [[COMPUTE]]
+        platform = levante
+        execution time limit = PT3M
+        [[[directives]]]
+            --wckey = RTW
+            --account = bk1088
+            --partition = interactive
+            --mem = 15G
```

Comment:
I have sent an e-mail to DKRZ support to ask why the memory needs to be so high (the jobs fail if the memory requested is less than this with messages like `slurmstepd: error: Detected 1 oom_kill event in StepId=<process-id>.batch. Some of the step tasks have been OOM Killed.`

Reply:
are you sure the process is not requesting a lot of memory? The actual use could be 4G but the process may request a lot more than that

Reply:
I have no idea! 😱 I realised though that I hadn't added in the actual usage, so I have done that in b17fa96. The values I have added here come from the ESMValTool logs, so I assume they're accurate? 🤪